src/users/api/controllers/createUser/CreateUserController.ts

```diff
@@ -0,0 +1,21 @@
+import { CreateUserUseCase } from '../../../useCases/createUser/CreateUserUseCase';
+import { Request, Response } from 'express';
+import { CreateUserDTO } from '../../dtos/CreateUserDTO';
+
+export class CreateUserController {
+  constructor(private readonly createUserUseCase: CreateUserUseCase) {}
+
+  async execute(request: Request, response: Response): Promise<void> {
+    const { email } = request.body;
+    const userDTO = new CreateUserDTO(email);
+
+    try {
+      const userResponse = await this.createUserUseCase.execute(userDTO);
+      response.status(201).json(userResponse);
+    } catch (err: unknown) {
+      response.status(500).json({
+        message: (err as Error).message,
```

Comment:
Can we have a logger to log those errors

Reply:
@thetnaingtta  Thet will add the logger module later right?